cmd/agent/app/integrations.go

```diff
@@ -115,6 +120,46 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func detectEnvironment() {
+	rootDir, _ = executable.Folder()
+	for {
+		agentReleaseFile := filepath.Join(rootDir, reqAgentReleaseFile)
+		if _, err := os.Lstat(agentReleaseFile); err == nil {
+			reqAgentReleasePath = agentReleaseFile
+			break
+		}
+
+		parentDir := filepath.Dir(rootDir)
+		if parentDir == rootDir {
+			break
+		}
+
+		rootDir = parentDir
+	}
+
+	pythonMajorVersion = config.Datadog.GetString("python_version")
+	constraintsPath = filepath.Join(rootDir, fmt.Sprintf("final_constraints-py%s.txt", pythonMajorVersion))
```

Comment:
We're not validating the existence of this file. If the file doesn't exist, we shouldn't set the constraintsPath, we should exit with an error.